Dockerfile

```diff
@@ -82,11 +86,12 @@ RUN /usr/bin/generate-keys --private-key matrix_key.pem && \
 
 ENV SERVER_NAME=localhost
 ENV API=0
+ENV COVER=0
 EXPOSE 8008 8448
 
 # At runtime, generate TLS cert based on the CA now mounted at /ca
 # At runtime, replace the SERVER_NAME with what we are told
 CMD /usr/bin/generate-keys --server $SERVER_NAME --tls-cert server.crt --tls-key server.key --tls-authority-cert /ca/ca.crt --tls-authority-key /ca/ca.key && \
     /usr/bin/generate-config -server $SERVER_NAME --ci > dendrite.yaml && \
     cp /ca/ca.crt /usr/local/share/ca-certificates/ && update-ca-certificates && \
-    /usr/bin/dendrite-monolith-server --really-enable-open-registration --tls-cert server.crt --tls-key server.key --config dendrite.yaml -api=${API:-0}
+    exec /complement-cmd.sh
```

Comment:
Running "complement" anything in this Dockerfile feels very very wrong. This is the base image for the entire server?

Reply:
Same as above, multi staged Dockerfile, this is the "complement" build target.

Reply:
Ripped it out, since the `complement-cmd.sh` would not have worked anyway.